intro-to-tests/practice-intro-to-pytest.md

```diff
@@ -0,0 +1,890 @@
+### !question
+
+Write a function to make the test pass. Use the information from the test to help determine what needs to be in the function body.
+
+```python
+def test_returns_true():
+    result = func_that_always_returns_true()
+    assert result
+```
+### !end-question
+### !placeholder
+
+```python
+def func_that_always_returns_true():
+    pass
+```
+### !end-placeholder
+### !tests
+```python
+import unittest
+from main import *
+
+class TestChallenge(unittest.TestCase):
+  def test_returns_true(self):
+      self.assertTrue(func_that_always_returns_true())
+```
+### !end-tests
+### !explanation
+
+An example of a working implementation:
+
+```python
+def func_that_always_returns_true():
+    return True
+```
+
+### !end-explanation
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 14 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: code-snippet
+* language: python3.6
+* id: 621ce40e-a848-4429-a100-76e6001f77c3
+* title: Intro to pytest
+##### !question
+
+Write a function to make the test pass. Use the information from the test to help determine what needs to be in the function body.
+
+```python
+def test_returns_true_if_odd():
+    number = 5
+    result = is_odd(number)
+    assert result
+```
+
+##### !end-question
+##### !tests
+
+```py
+import unittest
+from main import *
+
+class TestPython1(unittest.TestCase):
+    def test_returns_true_if_odd(self):
+        number = 5
+        result = is_odd(number)
+        self.assertTrue(result)
+```
+
+##### !end-tests
+##### !hint
+
+Logic to find that something is odd or even is best solved using the modulo operator.
+
+##### !end-hint
+##### !hint
+
+Don't forget to put your code in a function with the right name and parameters.
+
+##### !end-hint
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 15 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: code-snippet
+* language: python3.6
+* id: 71615b90-7562-480f-b868-50e23812a0f3
+* title: Intro to pytest
+##### !question
+
+Write a function to make the test pass. Use the information from the test to help determine what needs to be in the function body.
+
+```python
+def test_returns_false_if_even():
+    number = 6
+    result = is_odd(number)
+    assert not result
+```
+
+##### !end-question
+
+##### !tests
+
+```py
+import unittest
+from main import *
+
+class TestPython1(unittest.TestCase):
+    def test_returns_false_if_even(self):
+        number = 6
+        result = is_odd(number)
+        self.assertFalse(result)
+```
+
+##### !end-tests
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 16 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: code-snippet
+* language: python3.6
+* id: 386b0492-77de-4132-a81c-8746090296c2
+* title: Intro to pytest
+##### !question
+
+Write a function to make the test pass. Use the information from the test to help determine what needs to be in the function body.
+
+```python
+def test_returns_None_if_negative():
```

Comment:
If `None` is being capitalized here because it's a reserved word, then false and true should in the previous functions. But in deference to lowercase snake case, I'd expect this to be lowercase.
```suggestion
def test_returns_none_if_negative():
```